composer.json

```diff
@@ -1,36 +1,42 @@
 {
     "name": "spryker-eco/after-pay",
+    "type": "library",
     "description": "AfterPay module",
     "license": "MIT",
     "require": {
-        "php": ">=7.1",
-        "spryker/checkout": "^3.0.0 || ^4.2.0 || ^6.0.0",
-        "spryker/currency": "^2.0.0 || ^3.5.0",
-        "spryker/kernel": "^3.0.0",
-        "spryker/locale": "^3.0.0",
+        "php": ">=7.3",
+        "psr/http-message": "^1.0.0",
+        "spryker/checkout-extension": "^1.3.0",
+        "spryker/customer": "^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0",
+        "spryker/guzzle": "^2.3.0",
+        "spryker/kernel": "^3.30.0",
+        "spryker/log": "^1.0.0 || ^2.0.0 || ^3.0.0",
         "spryker/money": "^2.0.0",
-        "spryker/oms": "^7.0.0 || ^8.4.0 || ^10.0.0 || ^11.0.0",
         "spryker/payment": "^4.0.0 || ^5.0.0",
-        "spryker/propel-orm": "^1.0.0",
-        "spryker/quote": "^1.0.0 || ^2.6.0",
+        "spryker/refund": "^3.0.0 || ^4.0.0 || ^5.0.0",
         "spryker/sales": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
-        "spryker/session": "^3.0.0 || ^4.0.0",
         "spryker/step-engine": "^2.0.0 || ^3.0.0",
+        "spryker/store": "^1.1.0",
         "spryker/symfony": "^3.0.0",
         "spryker/util-encoding": "^2.0.0",
         "spryker/util-text": "^1.0.0",
         "spryker/zed-request": "^3.0.0"
     },
     "require-dev": {
+        "phpstan/phpstan": "*",
+        "spryker/application": "*",
+        "spryker/checkout": "^3.0.0 || ^4.2.0 || ^6.0.0",
         "spryker/code-sniffer": "*",
-        "spryker/country": "*",
-        "spryker/customer": "*",
+        "spryker/oms": "^7.0.0 || ^8.4.0 || ^10.0.0 || ^11.0.0",
```

Comment:
* We cannot move `Oms` and` Checkout` to this section because we have dependencies on them.

Reply:
I did this only because of Travis Validation job
I will revert this and show you